end_to_end/end_to_end_suite_test.go

```diff
@@ -1159,11 +1161,14 @@ var _ = Describe("backup and restore end to end tests", func() {
 				DROP EXTENSION test_ext3;
 			`)
 
-			timestamp := gpbackup(gpbackupPath, backupHelperPath,
-				"--metadata-only")
+			timestamp := gpbackup(gpbackupPath, backupHelperPath)
 			gprestore(gprestorePath, restoreHelperPath, timestamp,
 				"--redirect-db", "restoredb")
 
+			assertDataRestored(restoreConn, map[string]int{
+				"public.test1": 100,
+				"public.test2": 50,
```

Comment:
You added 100 rows to table `test2`, but you check only 50, why?

Reply:
because of filter in extension